### PR TITLE
Initial basic setup.

### PR DESCRIPTION
**Pull request by dbatiste**
Just to get rolling - linting and other things to come.

### CODE REVIEW

File: components/d2l-hello-world.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		};
+	}
+
+	constructor() {
+		super();
+		this.message = 'Hello World from LitElement';
+	}
+
+	render() {
+		return html`
```
Comment (svanherk) [inline components/d2l-hello-world.js]: So I think we did want to add that whitespace fix, since lit doesn't have a special way of handling it?

Comment (dbatiste) [reply components/d2l-hello-world.js]: I was looking at the source for the `TemplateResult` and I didn't see any reference to `strip-whitespace`.  I can try it out though.  Stay tuned.

Comment (dbatiste) [reply components/d2l-hello-world.js]: Ok - gave it a try and `setAttribute` is not on the returned `TemplateResult` object.  The TemplateResult does not contain DOM per se, but rather a description of what needs to be created.  I think the `html` template function that comes with Polymer must be a bit different than the one that comes from `lit-element`.

Comment (svanherk) [reply components/d2l-hello-world.js]: Ah ok, so we currently don't have a way to solve for that.  Should add that to the list!

**Pull request merged**